airflow/migrations/versions/e3a246e0dc1_current_schema.py

```diff
@@ -38,6 +38,8 @@
 depends_on = None
 
 
+print(COLLATION_ARGS)
```

Comment:
I guess we shoudl remove it ?

Reply:
Yes, that's for debug. I will revert the line.